dm_cli/bin/dm

```diff
@@ -216,6 +218,36 @@ def import_packages(context, path: str, data_source: str):
                 exit(1)  # If any of the parallel functions did not return True, they fail, and so we exit with 1
 
 
+@pkg_cli.command("export")
+@click.argument("absolute_document_reference", required=True)
+@click.argument("export_location", required=False, default="")
+@click.option("-u", "--unpack", is_flag=True, help="Flag to determine if the downloaded zip should be unpacked.")
+@click.option("-o", "--overwrite", is_flag=True, help="If flag is set, existing folder/document with the same name as the downloaded folder/document will be overwritten.")
```

Comment:
Don't feel like this is properly named.
Is not realy a overwrite or not.
Looks like it more of a "replace in place" or "export with a generated uuid in name".

I'd say we either rename, or rewrite to overwrite if flag is there, and exit with warning if not

Reply:
Personally, I think the flag name makes sense.


- perhaps we can get opinions from @eoaksnes , @netr0m  or Babak :)

Reply:
removed overwrite flag - now, command will exit if path of exported document/folder already exists on local disk.